sysutils/jq/Portfile

```diff
@@ -3,22 +3,22 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        stedolan jq 1.6 jq-
+github.setup        jqlang jq 1.6 jq-
+github.tarball_from releases
 revision            4
+
 categories          sysutils
-platforms           darwin
+license             MIT
 maintainers         {raimue @raimue} \
                     openmaintainer
-license             MIT
 
 description         A lightweight and flexible command-line JSON processor
 long_description\
     jq is like sed for JSON data – you can use it to slice and filter and \
     map and transform structured data with the same ease that sed, awk, \
     grep and friends let you play with text.
 
-homepage            https://stedolan.github.io/jq/
-github.tarball_from releases
+homepage            https://jqlang.github.io/jq
```

Comment:
When you get the opportunity, the slash at the end of this URL should be restored. GitHub Pages directory URLs without a slash get redirected to have a slash, so we should use that canonical with-a-slash URL in the Portfile.

Reply:
Done in https://github.com/macports/macports-ports/commit/4946089fa18b57e1d6fd1e138307d293f44786c2